.github/workflows/build.yml

```diff
@@ -61,22 +61,32 @@ jobs:
           # Install packages from Homebrew
           brew bundle install
 
+      # `autopoint`, `gperf` tools needed for another package, probably wayland dependencies
       - name: Install Linux dependencies
         if: ${{ matrix.os.preset == 'linux' }}
         run: |
           sudo apt update
           sudo apt install -y --no-install-recommends \
-            ninja-build cmake g++ libgtest-dev libsdl2-dev libncurses-dev zlib1g-dev libspdlog-dev
+            ninja-build cmake g++ libgtest-dev libsdl2-dev libncurses-dev zlib1g-dev libspdlog-dev \
```

Comment:
I can't remember if I asked before but libsdl2-dev, zlib & spdlog should now be provided by VCPKG ?